dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -63,3 +63,6 @@ trustStorePassword.desc=The password that is used to load the truststore file. T
 
 trustStoreType=Truststore type
 trustStoreType.desc=The keystore type for the truststore. Supported types are JKS, PKCS12 and JCEKS.
+
+renewBeforeExpiration=Renew time before expiration
+renewBeforeExpiration.desc=The amount of time before the expiration date of the certificate to request a new certificate. The certificate renew request continues until a new certificate is received. For example, if the renewBeforeExpiration property is set to seven days, the ACME service requests a new certificate seven days before the expiration date of the current certificate.
```

Comment:
Didn't we say that <= 0 we would disable renewal? Or was I thinking of something else?

Reply:
Also, this really is the time before the expiration date of the certificate where acme will make the first attempt to request a new certificate from the ACME CA server. If it fails, we will continue to try and request a new one every X amount of time until we are able to get a new certificate.

Reply:
Reworded.